cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.md

```diff
@@ -56,9 +56,17 @@ Use of `XercesDOMParser` do this to prevent XXE:
 
 ``` cpp
 XercesDOMParser *parser = new XercesDOMParser;
-parser->setCreateEntityReferenceNodes(false);
+parser->setCreateEntityReferenceNodes(true);
 ```
 
+Creating reference nodes prevents XXE coming from DTDs, local file inclusions as well as outbound http requests.
```

Comment:
Having just this does allow outbound HTTP - you need `setDisableDefaultEntityResolution(true)` as well to block it.